index.bs

```diff
@@ -10615,21 +10615,20 @@ operation such as resolution of {{Promise}}s in the {{AudioWorkletGlobalScope}}.
 				2. [=Visit=] <var>node</var>.
 
 				<div algorithm="visiting a node">
-					<dfn lt="Visit">Visiting a node</dfn> <var>node</var> means performing
+					<dfn lt="Visit">Visiting a node</dfn> means performing
```

Comment:
Since this step includes the recursion, I think it's better to keep it as a separate algorithm.